plugins/scaffolder/src/components/fields/EntityPicker/EntityPicker.tsx

```diff
@@ -58,6 +59,7 @@ export const EntityPicker = ({
       error={rawErrors?.length > 0 && !formData}
     >
       <Autocomplete
+        id={idSchema && idSchema.$id}
```

Comment:
```suggestion
        id={idSchema?.$id}
```
Should preferably use optional chaining here and below, I think.